src/components/Form.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from "react";
+
+class Form extends Component {
```

Comment:
Is this a general form? If not the name should reflect what exactly it is doing.